README.md

```diff
@@ -45,7 +45,7 @@ Then add the Maven dependency to your project.
     <dependency>
         <groupId>uk.gov.service.notify</groupId>
         <artifactId>notifications-java-client</artifactId>
-        <version>3.4.0-RELEASE</version>
+        <version>3.5.0-RELEASE</version>
```

Comment:
I would consider changing the version to 3.6.0-RELEASE here, too.
The same goes for line 63 (in the Gradle section below).